### ISSUE TITLE
subtitle-search returns duplicate results

### ISSUE THREAD

**Issue by mrfragger** [zenwarr/mpv-config#15]
I got so excited by all the highlighting fixes you did I failed to notice it's returning the results twice.

![Screen Shot 2023-10-05 at 10 05 59 PM](https://github.com/zenwarr/mpv-config/assets/80211672/22fc7329-dfc8-4d12-9b3e-bad0982d75bd)

**Comment by zenwarr** [comment]
Sorry, but I cannot reproduce this behavior. Is it duplicated with any subtitle you use, or only with some specific files?

**Comment by mrfragger** [comment]
yea it's with all of them.  vtt even if I convert to srt still happens 
ffmpeg -i some.vtt some.srt

ok sorry I figured it out...it's the visualizer.lua script that causes it to return duplicate results.
https://github.com/mfcc64/mpv-scripts/blob/master/visualizer.lua

I'll try to figure out why.

ok it seems if the original script is there it doesn't cause duplicates but because on a Mac I had to comment out the following code otherwise when it toggles to the blank screen mpv completely crashes.

```
  -- local hasvideo = false
        -- for id, track in ipairs(mp.get_property_native("track-list")) do
        --     if track.type == "video" then
        --         hasvideo = true
        --         break
        --     end
        -- end
```
Ultimately it just means I can't use visualizer.lua
